### PR TITLE
Added injectable okhttp networking module

### PR DESCRIPTION
**Pull request by shoab10**
- Added Stetho interceptor to debug network via Stetho

### CODE REVIEW

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -1,18 +1,31 @@
 <?xml version="1.0" encoding="utf-8"?>
-<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (AgrimAsthana-CS) [inline app/src/main/res/layout/activity_main.xml]: Thank you very much for doing this !

**Comment by AgrimAsthana-CS** [comment]
LGTM even tough it already is merged

**Comment by agrimrules** [comment]
LGTM 👍  but its already merged ?

**Comment by shoab10** [comment]
yep i merged it.

**Comment by shoab10** [comment]
yes .

**Pull request merged**